DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -32,6 +32,21 @@ databricksClusterNodeType=Standard_DS3_v2
 #Databricks SKU
 databricksSku=premium
 
+# HDInsight cluster version. Please use a whitelisted subscriptionId if you want to deploy version 4.0 (for now).
```

Comment:
Probably better not to use "for now" unless there is any action item on our plate.

Reply:
removed the inappropriate comment